source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,146 @@
+ - Added a setting to always land users at the newest messages in a channel via **Account settings > Advanced > Scroll position when viewing an unread channel**.
+ - Added email headers to notification emails so they can be threaded by email clients.
+ - Added **Save** and **Cancel** buttons for post inline editing.
+ - Enterprise trial details are now displayed for end users in the product switcher menu.
+ - Updated the Edit Header modal text description to be applicable to channels, direct messages, and group messages.
+ - Added the ability to quickly and easily forward posts as permalinks with their respective permalink previews.
+ - Added a red destructive action color to ``Archive Channel`` and ``Leave Channel`` menu actions.
+ - Plugin activation errors now show in the plugin management page and marketplace.
+ - Added accessibility to the emoji picker skin tone selector and reversed the order of the skin tone selections in the emoji selector.
+
+#### Administration
+ - Added an "Upgrade" button for Admins on the navigation bar.
+ - Added the ability for Admins to quickly view different paid license options inside the product.
+ - Added the ability to start a trial from the **Invite People** modal.
+ - Added the ability for end users to notify Admins to upgrade their workspace.
+ - Admins are now able to search for channel IDs via **System Console > User Management > Channels** page.
+ - In the **System Console** left-hand side, paid features icons are now displayed on the menu entries to indicate enterprise features.
+ - Added ``webSocketClient`` to ``Pluggable`` and ``PostWillRenderEmbed`` plugin registered components.
+ - Added a new static system-level role called Custom Group Manager. This role has permissions to create, edit, and delete custom user groups via User Groups in the Products menu. It can be used to assign individual users this ability when Custom Groups permissions are removed for All Members via the **System Console** (**System Console > Permissions > Edit Scheme > Custom Groups**).
+ - Export file names now contain the ID of the job they were generated by.
+
+### Performance
+ - Removed ``getLastPostPerChannel`` selector for improved performance in channel sorting.
+
+### Bug Fixes
+ - Fixed an issue with pasting a GitHub code snippet in the message box when text is selected.
+ - Fixed an issue where fully typed emojis that contained a capital letter were not correctly displayed.
+ - Fixed an issue where the archived icon did not display correctly in dark themes.
+ - Fixed an issue where password requirements were not enforced when Development Mode was enabled.
+ - Fixed an issue where users were able to attempt to edit the channel header of an archived channel on the right-hand side.
+ - Fixed an issue where the “Your Trial Ended” banner hid the product switcher menu.
+ - Fixed an issue where the custom status date format was not set to ``yyyy-MM-dd``.
```

Comment:
Question: Is MM intended when the rest of the date format is in lowercase (yyyy dd)?